index.js

```diff
@@ -1,46 +1,63 @@
 /*! MIT License © Sindre Sorhus */
 
-const getGlobal = property => {
-	/* istanbul ignore next */
-	if (typeof self !== 'undefined' && self && property in self) {
-		return self[property];
-	}
+const globals = {};
 
-	/* istanbul ignore next */
-	if (typeof window !== 'undefined' && window && property in window) {
-		return window[property];
-	}
+{
+	const getGlobal = property => {
+		/* istanbul ignore next */
+		if (typeof self !== 'undefined' && self && property in self) {
+			return self[property];
+		}
 
-	if (typeof global !== 'undefined' && global && property in global) {
-		return global[property];
-	}
+		/* istanbul ignore next */
+		if (typeof window !== 'undefined' && window && property in window) {
+			return window[property];
+		}
 
-	/* istanbul ignore next */
-	if (typeof globalThis !== 'undefined' && globalThis) {
-		return globalThis[property];
+		if (typeof global !== 'undefined' && global && property in global) {
+			return global[property];
+		}
+
+		/* istanbul ignore next */
+		if (typeof globalThis !== 'undefined' && globalThis) {
+			return globalThis[property];
+		}
+	};
+	
+	const globalProperties = [
+		'document',
+		'Headers', 
+		'Request',
+		'Response',
+		'ReadableStream',
+		'fetch',
+		'AbortController',
+		'FormData'
+	];
+
+	const props = {};
+	for (const property of globalProperties) {
+		props[property] = {
+			get() {
+				return getGlobal(property);
+			}
+		};
 	}
-};
 
-const document = getGlobal('document');
-const Headers = getGlobal('Headers');
-const Request = getGlobal('Request');
-const Response = getGlobal('Response');
-const ReadableStream = getGlobal('ReadableStream');
-const fetch = getGlobal('fetch');
-const AbortController = getGlobal('AbortController');
-const FormData = getGlobal('FormData');
+	Object.defineProperties(globals, props);
+}
 
 const isObject = value => value !== null && typeof value === 'object';
-const supportsAbortController = typeof AbortController === 'function';
-const supportsStreams = typeof ReadableStream === 'function';
-const supportsFormData = typeof FormData === 'function';
+const supportsAbortController = typeof globals.AbortController === 'function';
+const supportsStreams = typeof globals.ReadableStream === 'function';
+const supportsFormData = typeof globals.FormData === 'function';
 
 const deepMerge = (...sources) => {
 	let returnValue = {};
 
 	for (const source of sources) {
 		if (Array.isArray(source)) {
-			if (!(Array.isArray(returnValue))) {
+		  if (!(Array.isArray(returnValue))) {
```

Comment:
```suggestion
			if (!(Array.isArray(returnValue))) {
```

Reply:
![image](https://user-images.githubusercontent.com/36894700/64478531-fb75f280-d1a9-11e9-9063-d363e4296989.png)

but I can edit the file directly... hmm...